sqldb/sqlc/migrations/000009_payments.up.sql

```diff
@@ -2,43 +2,12 @@
 -- Payment System Schema Migration
 -- ─────────────────────────────────────────────
 -- This migration creates the complete payment system schema including:
--- - Payment intents (BOLT 11/12 invoices, offers)
+-- - Payment intents (only BOLT 11 invoices for now)
 -- - Payment attempts and HTLC tracking
 -- - Route hops and custom TLV records
 -- - Resolution tracking for settled/failed payments
 -- ─────────────────────────────────────────────
 
--- ─────────────────────────────────────────────
--- Payment Intents Table
```

Comment:
nit: for future commits, it'd be easier for the reviewers if we don't move the code yet add some changes in one commit, which can be challenging for the reviewers to do a line-by-line comparison.